rpc-client-api/src/response.rs

```diff
@@ -549,3 +551,73 @@ pub struct RpcPrioritizationFee {
     pub slot: Slot,
     pub prioritization_fee: u64,
 }
+
+#[cfg(test)]
+pub mod tests {
+
+    use {
+        super::RpcPerfSample,
+        serde::Serialize,
+        serde_json::{self, json},
```

Comment:
Shouldn't need `self` here

Reply:
In addition, it is pretty common to have `use::super::*` here in the `tests` mod. Logically, it makes sense that tests for a component will need lots of the same deps as the source it is testing. The clippy lint even has an exception for this exact case:
https://rust-lang.github.io/rust-clippy/master/#wildcard_imports

Reply:
> Shouldn't need `self` here

I'm using `serde_json::from_str()` and `serde_json::to_value()`.
Without an explicit qualification, `from_str()` and `to_value()` seems overly generic.

> it is pretty common to have `use::super::*` here in the `tests` mod.

As for the `use super::*` - I personally prefer explicit imports.
But if this is an agreed upon style – I'll change it.
There are downsides to `use super::*` as it pulls more than necessary.
In particular, it imports all the symbols imported into the parent module.
I've written a bit more details in the Discord in `#devops`.

Reply:
Because it is a crate defined as a dependency in Cargo.toml, the top level `serde_json` is already being imported. You can just remove `self` here and leave the calling code as-is. (I am definitely not suggesting that you import `serde_json::{from_str, to_value}` and remove the crate name from the calling sites.)

Reply:
Isn't that confusing, that some imports are explicit and some are implicit? :P
Done.